crypto/x509/x509_vfy.c

```diff
@@ -974,9 +994,11 @@ static int check_revocation(X509_STORE_CTX *ctx)
 {
     int i = 0, last = 0, ok = 0;
 
-    if ((ctx->param->flags & X509_V_FLAG_CRL_CHECK) == 0)
+    if (!(ctx->param->flags & X509_V_FLAG_CRL_CHECK)
```

Comment:
style nit: please keep explicit test against 0
```
if ((ctx->param->flags & X509_V_FLAG_CRL_CHECK) != 0
    && (ctx->param->flags & X509_V_FLAG_OCSP_CHECK) !=0)
```

Reply:
Yes, but the semantics must not change.
Correction and simplification of your suggestion: `(ctx->param->flags & (X509_V_FLAG_CRL_CHECK | X509_V_FLAG_OCSP_CHECK))  == 0)`